src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -208,6 +223,8 @@ class CoreWorkerDirectTaskSubmitter {
   absl::flat_hash_map<NodeID, std::shared_ptr<WorkerLeaseInterface>> remote_lease_clients_
       GUARDED_BY(mu_);
 
+  absl::flat_hash_map<TaskID, Status> taskID2status;
```

Comment:
hmm does this map grow infinitely?

Reply:
I am thinking of removing this loop stuff in C++ code and instead place it in `_raylet.pyx`.

Reply:
This map is removed.